### DISCUSSION TITLE
[Envoy API Gateway] Rate limit by specific route

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#526] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

v13.0.5

### Description

Hi guys,
I just configured a sample `/hello` route on a demo service and I noticed that, using Envoy, you can specify the endpoint rate limit (req/s).

I saw it's not possible to specify the rate limit for sub-routes too:
<img width="1172" alt="Screenshot 2024-07-26 at 11 36 53" src="https://github.com/user-attachments/assets/0db720d5-51f3-4be5-851f-2002e450d0f4">

Is that an Envoy limitation (I think it's not) or the Console just doesn't supports this feature yet?
If needed, the only way to achieve this at the moment is through the Envoy Advanced Configurations?

Thanks.

**Comment by davidebianchi** [comment, marked as answer]
Hi @silversoul93! It is a Console limitation, it could be added in the future. In the meantime, you could add a new endpoint `/basepath/hello-1` endpoint, and track on it the rate limit. Should this work for you?

**Reply by frengooo** [reply]
This should work, but I'll need to do that in a project with several endpoints 😅 it will be uncomfortable, but it's ok for now, thanks @davidebianchi
